elastic/kibana
# [Infra Monitoring UI] Change infrastructure pages titles to use breadcrumbs #135874

## Summary
Related https://github.com/elastic/kibana/issues/135874
This PR changes the titles in Logs and Infrastructure apps. Similar to other apps now the titles are constructed from the breadcrumbs in reverse order. This helps us to keep the titles consistent. Also, it makes it easier for users to navigate using screen readers.

Based on the comment regarding how we should set the document title [here](https://github.com/elastic/kibana/issues/135760#issuecomment-1252074778) I also changed the error page title to use `chrome.docTitle.change`  in ([ebe26be](https://github.com/elastic/kibana/pull/140824/commits/ebe26be0dbbb63ba93e294661810d228f657411c)). The error page can also use the general breadcrumbs title but it has a different title now. So I kept the current title and added the Observability title part to it so it will be closer to the other page titles.

## Example
Old title: Metrics | Inventory - Kibana
New title: Inventory - Infrastructure - Observability - Elastic

## Testing
I checked locally. Steps:
- Navigate to the Infrastructure page
- Check all document titles (all pages: Hosts/Metrics Explorer/Inventory and sub-pages)
- Same with Logs pages (Stream/Categories/Anomalies)
✅  check in the cloud as well.
